strategies/SQFSuperFluidStrategy/Base/spec.ts

```diff
@@ -0,0 +1,135 @@
+import { Address, BeforeAll, BigInt, Event, LiveTable, OnEvent, Property, Spec } from '@spec.dev/core'
+
+import { decodeSQFSuperFluidInitializedData } from "../../../shared/decoders.ts";
+
+/**
+ * SQFSuperFluids details
+ */
+@Spec({
+    uniqueBy: ['chainId', 'poolId']
+})
+class SQFSuperFluids extends LiveTable {
```

Comment:
I'll be trying to set a design pattern standard that keeps all live table class names singular